README.md

```diff
@@ -93,45 +93,80 @@ Here's an overview of the production setup (the focus is on Next.js, therefore d
 
 ![image](https://user-images.githubusercontent.com/4640135/203581627-82ab19ca-7de7-4343-ae05-2a4f6330f38a.png)
 
-### Tests
+## Tests
 
-We use Jest and Playwright (with MSW) to run our tests.
+We use [Jest](https://jestjs.io) and [Playwright](https://playwright.dev/) (with [MSW](https://mswjs.io/)) to run our tests.
 
-You can use any of the following commands to run Playwright tests:
+At the moment, we only have tests for the `frontend`. These tests are located in the [`frontend/tests`](frontend/tests) directory.
```

Comment:
For the directory hyperlinks, does GitHub know to direct to them via this repo or should they be actual links?

e.g. https://github.com/iFixit/react-commerce/tree/main/frontend/tests instead of /frontend/tests

Reply:
Yes, you can go to https://github.com/iFixit/react-commerce/tree/update-test-docs to see the full render of the `README`

Reply:
https://user-images.githubusercontent.com/22064420/225376798-86247786-0fb8-4210-8dc4-36f41041e787.mp4

Reply:
TIL